### PR TITLE
add osx support through pyobjus

### PR DESCRIPTION
**Pull request by tshirtman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,5 +29,5 @@ If you install it from sources:
 
 **Output**:
 
->monitor(1920x1080+1920+0)  
+>monitor(1920x1080+1920+0)
```
Comment (rr-) [inline README.md]: Trailing whitespace is significant in Markdown files

**Comment by tshirtman** [comment]
fixed, sorry for that.

**Comment by rr-** [comment]
Thanks!

**Comment by fragofer** [comment]
Hi, thanks for the great work you have done with this package. I'd like to know which release number has this code for support osx screens.

Thanks

**Comment by rr-** [comment]
0.2.2+

**Pull request merged**